### DISCUSSION TITLE
Keyboard shortcuts cheatsheet

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#412] [Guides: Atri Editor]
## Windows & Linux OS

| Action                            | Key                               | References                               | 
| ------------------------------- | --------------------------------------- | --------------------------------------- |
| Delete the selected component             | `Delete`                  | -                |
| Copy and paste a component      | `Ctrl + C` -> `Ctrl + V`                  | [Video link](https://youtu.be/y8xLjOc-pJk)                 |
| Undo a change | `Ctrl + Z`                   | -              |
| Build and Run     | `Ctrl + B `                 | -              |
| Select a parent component      | Select child -> `Up Arrow`                   | -              |
| Reposition a component      | Select component  ->  `Ctrl`                 | [Video link](https://www.youtube.com/watch?v=Lrcd0I8dgQU)             |

## MacOS

| Action                            | Key                               | References                               | 
| ------------------------------- | --------------------------------------- | --------------------------------------- |
| Delete the selected component             | `Delete`                  | -                |
| Copy and paste a component      | `Cmd + C` -> `Cmd + V `                 | [Video link ](https://youtu.be/y8xLjOc-pJk)                |
| Undo a change | `Cmd + Z`                   | -              |
| Build and Run      | `Cmd + B`                  | -              |
| Select a parent component      | Select child -> `Up Arrow`                   | -              |
| Reposition a component      | Select component  ->  `Cmd`                 | [Video link](https://www.youtube.com/watch?v=Lrcd0I8dgQU)             |

**Comment by Aman-Joshi989** [comment]
how to use color gradient in atri

**Reply by anniwalvekar** [reply]
follow this tutorial- https://youtu.be/aL07qPxNOLU

**Comment by ramchaudhary2000** [comment]
background: rgb(186,75,75);
background: linear-gradient(90deg, rgba(186,75,75,1) 26%, rgba(177,176,176,1) 52%, rgba(231,31,31,1) 83%);

change the color value by your requirement
